Blog/GamingGuruBlog.Web/GamingGuruBlog.Web/UIConverter.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using GamingGuruBlog.Domain;
+using GamingGuruBlog.Domain.Interfaces;
+using GamingGuruBlog.Domain.Models;
+using GamingGuruBlog.Web.Models;
+using System.Web.Mvc;
+
+namespace GamingGuruBlog.Web
+{
+    public class UIConverter
```

Comment:
All this class should do is convert a viewmodel to a domain model and visa versa.